content/applications/general/multi_company.rst

```diff
@@ -0,0 +1,114 @@
+   Francisco), while My Company (Chicago) is also active.
+
+.. note::
+   The database may refresh after each checkbox is ticked.
+
+.. _general/active-companies:
+
+Multiple active companies
+-------------------------
+
+If more than one company is active at a time, one company is highlighted in purple, and is listed on
+the menu bar. This is the considered the *current* company.
+
+When creating a new record, the *current* company is added to the record in the :guilabel:`Company`
+field, except under the following circumstances:
+
+- The :guilabel:`Company` field for a new product or a new contact is left blank.
+- If there is a related document already in the system, the :guilabel:`Company` field on the new
+  record defaults to the same company.
+
+.. example::
+   Mitchell Admin has multiple companies enabled, but the active company is `My Company (Chicago)`.
+   When he creates a new product record, the :guilabel:`Company` field is left blank by default.
+
+   When he creates a new sales team, the :guilabel:`Company` field automatically defaults to `My
+   Company (Chicago)`.
+
+Sharing data
+============
+
+In a |mcd|, certain records are able to be utilized by all of the companies (or several, based on
+permissions).
+
+Products
+--------
+
+In an |mcd|, new products are created with the :ref:`company field <general/active-companies>`
+blank, by default. If the field remains blank, the product is shared across all companies.
+:ref:`company field <general/active-companies>`
+
+Contacts
+--------
+
+Similar to products, contact records are shared across companies, by default. To limit access to a
+single company, click the :ref:`company field <general/active-companies>` and select a company to
+assign the contact to.
+
+Inter-company transactions
+==========================
+
+The :ref:`Inter-Company Transactions <general/inter-company>` feature allows for one company in the
+database to sell or purchase goods and services from another company within the same database.
+Counterpart documents for orders and invoices can be automatically generated and synchronized,
+depending on the configuration settings.
+
+.. warning::
+   To ensure inter-company transactions are handled appropriately, certain configurations, such as
+   fiscal positions and localizations, need to be accurately assigned. See :ref:`Inter-Company
+   Transactions <general/inter-company>` for additional information.
+
+Use cases
+=========
+
+Use case #1
+-----------
+
+A multinational retail chain, which operates in both the United States and Canada, needs to manage
+transactions in both USD and CAD currencies. Additionally, because both countries have different tax
+laws and regulations, it is in the best interest of the customer to utilize the multi-company
+feature. This allows for inter-company transactions that they will need to manage inventory moves
+across international boarders, while making it simple to sell to customers in both countries in
+their own currency.
```

Comment:
```suggestion
A multinational retail chain, which operates in the United States and Canada, needs to manage
transactions in both USD and CAD currencies. 

Additionally, because both countries have different tax laws and regulations, it is in the best interest
of the customer to utilize the multi-company feature. 

This allows for inter-company transactions they need in order to manage inventory moves across international borders, while making it simple to sell to customers in both countries in their own currency.
```